addons/trenches/functions/fnc_initTrench.sqf

```diff
@@ -0,0 +1,35 @@
+#include "script_component.hpp"
+/*
+ * Author: chris579, Salbei
+ * Inits a trench
+ *
+ * Arguments:
+ * 0: trench <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [TrenchObj] call ace_trenches_fnc_initTrench
+ *
+ * Public: No
+ */
+
+params [
+    ["_object", objNull, [objNull]]
+];
+if (isServer) then {
+   _object setVariable [QGVAR(progress), 1, true];
+};
+
+if (is3DEN) exitWith {
+    [_object] call FUNC(initTrench3DEN);
+};
+
+if (local _object) then {
+    // Has to be spawned to ensure MP compatibility
```

Comment:
Explain?
Only problem I see is trying to call setObjectTexture in same frame as createVehicle while the vehicle isn't fully initialized yet.
How about just a execNextFrame here?

Reply:
Not my code, its from Chris579. I do not know why you need it.